.github/workflows/ElastixGitHubActions.yml

```diff
@@ -1,7 +1,7 @@
 name: Elastix
 
 on: [push, pull_request]
-  
+
```

Comment:
Preferably don't modify  .github/workflows/ElastixGitHubActions.yml with this pull request, as it seems unrelated.

Reply:
Thanks, I see you addressed my comment on ElastixGitHubActions.yml 👍